src/lib/mousetrap-record.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * This extension allows you to record a sequence using Mousetrap.
```

Comment:
The thing is ccampbell repo seems not to be maintained and I simply missed this fresh fork. The best thing they updated [the plugin](https://github.com/mousetrap-js/mousetrap/blob/master/plugins/record/mousetrap-record.js) with the features we need!

They implemented `stopRecord` function and I can pass `recordSequence=false` to remove the delay after the first key is recorded. I will reimplement key recoding using their library and will add their repository to bower.